frontend/public/components/utils/kebab.tsx

```diff
@@ -48,7 +48,11 @@ const KebabItemEnabled: React.FC<KebabItemProps> = ({ option, onClick, onEscape,
 };
 
 const KebabItemDisabled: React.FC<KebabItemDisabledProps> = ({ option }) => {
-  return <button className="pf-c-dropdown__menu-item pf-m-disabled">{option.label}</button>;
+  return (
+    <button className="pf-c-dropdown__menu-item pf-m-disabled" data-test-action={option.label}>
```

Comment:
It seems like this button should have a `disabled` attribute for accessibility, but I don't know if that will change the appearance. Ideally we'd just find the menu item and check if it has the disabled attribute instead of checking the `pf-m-disabled` class.

Reply:
Hi @spadgett , to truly disable the button and rewrite alert-routing-modal to use `useState` is moving this PR beyond just integration tests.  I also have two alertmanager follow up PRs; once of which is a big refactor.  Would be be ok to resolve these last two comments in a follow up PR so I can avoid a lot of merge conflicts?  -thanks

Reply:
> to truly disable the button and rewrite alert-routing-modal to use useState is moving this PR beyond just integration tests. 

Right, please don't refactor the existing code :) We shouldn't do any refactors past feature freeze. I was just making an observation for the future.